utils/standalone_searx.py

```diff
@@ -1,5 +1,74 @@
 #!/usr/bin/env python
+"""Script to run searx from terminal.
 
+Import searx directly will raise error.
+User have to set `SEARX_DEBUG` env or searx's secret key to pass that error
+
+>>> import searx
+ERROR:searx:server.secret_key is not changed. Please use something else instead of ultrasecretkey.
+NameError: name 'exit' is not defined
+>>> from os import environ
+... environ.setdefault('SEARX_DEBUG', 'true')
+... import searx
+
+Getting categories without initiate the engine will only return `['general']`
+
+>>> import searx.engines
+... list(searx.engines.categories.keys())
+['general']
+>>> import searx
+>>> import searx.engines
+... searx.engines.initialize_engines(searx.settings['engines'])
+... list(searx.engines.categories.keys())
+['general', 'it', 'science', 'images', 'news', 'videos', 'music', 'files', 'social media', 'map']
+
+Example to use this script:
+
+.. code::  bash
+
+    $ export SEARX_DEBUG=1 && python3 utils/standalone_searx.py rain
```

Comment:
You can remove the `export` and the `&&`

FYI: Here is how I tested:

```bash
$ make install # this will also build the 'buildenv' target
$ source ./local/py3/bin/activate
(py3) $ SEARX_DEBUG=1 python utils/standalone_searx.py rain
````
```